packages/compiler/src/index.ts

```diff
@@ -210,6 +210,7 @@ export {
 export {
   CompileResult,
   createServer,
+  CustomCompileResult,
```

Comment:
this is currently too taylored for the typespec-vscode use case and doesn't make sense to be exposed as a core export. Move it to the internals until we have a universal solution for compiler stats

Reply:
@timotheeguerin thanks. move `CustomCompileResult` to internal export